tests/integration/tests/conftest.py

```diff
@@ -0,0 +1,47 @@
+"""
+Integration tests setup configurations and fixtures
+"""
+import time
+from pathlib import Path
+import pytest
+from commonlib.io_utils import get_k8s_yaml_objects
+
+
+DEPLOY_YML = "../../deploy/cloudbeat-pytest.yml"
+
+
+@pytest.fixture(scope='module', name='start_stop')
+def fixture_start_stop(k8s, api_client, cloudbeat_agent):
```

Comment:
I think the name could be more indicative, something like `start_cloudbeat`.

Reply:
This fixture performs also start and stop cloudbeat, therefore if we need more indicative name it may be fixture_start_stop_cloudbeat